js/view.js

```diff
@@ -2,9 +2,10 @@ var projectView = {};
 
 projectView.handleMainNav = function() {
   $('.main-nav').on('click', '.tab', function(){
-    console.log($(this).data('content'));
-    $('.tab-content').hide();
-    $("#" + $(this).data('content')).show();
+    $('#about').hide();
+    $('#main-page').hide();
```

Comment:
Are there any similarities between main and about that would let you hide them both at the same time? HTML elements, maybe? Or a class you could add?